core/src/main/java/io/neow3j/protocol/Neow3j.java

```diff
@@ -131,59 +127,40 @@ public Hash160 getNNSResolver() {
     }
 
     /**
-     * Gets the executor service used for polling new blocks from the Neo node.
-     * <p>
-     * The default executor service is a {@link ScheduledThreadPoolExecutor} with as many threads as CPUs available
-     * to the JVM.
+     * Sets the NeoNameService script hash that should be used to resolve NNS domain names.
      *
-     * @return the configured executor service.
+     * @param nnsResolver the NNS resolver script hash.
      */
-    public ScheduledExecutorService getScheduledExecutorService() {
```

Comment:
~as far as I know, this is required for tests. Can you please check?~

Ah, ok, I've seen you didn't delete this. Just shuffled the code around. We still have the `getScheduledExecutorService()`.

Reply:
Correct, just reordered.